src/shared/lib/StoreUtils.ts

```diff
@@ -65,22 +67,19 @@ import { indexPdbAlignments } from 'shared/lib/PdbUtils';
 import { IGisticData } from 'shared/model/Gistic';
 import { IMutSigData } from 'shared/model/MutSig';
 import {
+    AlterationTypeConstants,
     CLINICAL_ATTRIBUTE_ID_ENUM,
     DataTypeConstants,
     GENOME_NEXUS_ARG_FIELD_ENUM,
 } from 'shared/constants';
-import {
-    AnnotatedNumericGeneMolecularData,
-    CustomDriverNumericGeneMolecularData,
-} from '../../pages/resultsView/ResultsViewPageStore';
-import { AlterationTypeConstants } from 'shared/constants';
+import { AnnotatedNumericGeneMolecularData } from '../../pages/resultsView/ResultsViewPageStore';
```

Comment:
would like to avoid importing things from store in utils file.  This seems like it's a type so belongs in model file.

Reply:
Move exported types and interfaces from ResultViewPageStore to ./model in [b64b0c0](https://github.com/cBioPortal/cbioportal-frontend/pull/4552/commits/b64b0c00389c30a4cc4f51e7ed12968ad17bcbe9)